### PR TITLE
Micah riley

### PR DESCRIPTION
**Pull request by micahriley88**

### CODE REVIEW

File: Article/Article.js [hunk 1]
```diff
@@ -112,3 +112,44 @@ const data = [
   Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new article.
 
 */
+
+const articles = document.querySelector('.articles')
+
+data.forEach(data => {
+  articles.appendChild(createArticle(data.title, data.date, data.firstParagraph, data.secondParagraph, data.thirdParagraph));
+})
+
```
Comment (dustinsnoap) [inline Article/Article.js]: ```//what does this function do?```

File: Article/Article.js [hunk 2]
```diff
@@ -112,3 +112,44 @@ const data = [
...
+function createArticle(title, date, firstParagraph, secondParagraph, thirdParagraph) {
+  const article = document.createElement('article')
+  const h2 = document.createElement('h2');
+  const dateText = document.createElement('p');
+  const p1 = document.createElement('p');
+  const p2 = document.createElement('p');
+  const p3 = document.createElement('p');
+  const button = document.createElement('span')
+
```
Comment (dustinsnoap) [inline Article/Article.js]: ```//what is this section of the function doing?```

File: Article/Article.js [hunk 3]
```diff
@@ -112,3 +112,44 @@ const data = [
...
+  article.appendChild(p1);
+  article.appendChild(p2);
+  article.appendChild(p3);
+  article.appendChild(button);
+
+  article.classList.add('article');
+  dateText.classList.add('date');
+  button.classList.add('expandButton');
+
+
```
Comment (dustinsnoap) [inline Article/Article.js]: ```//What about this section?```

File: Article/Article.js [hunk 4]
```diff
@@ -112,3 +112,44 @@ const data = [
...
+  h2.textContent = title;
+  dateText.textContent = date;
+  p1.textContent = firstParagraph;
+  p2.textContent = secondParagraph;
+  p3.textContent = thirdParagraph;
+  button.textContent = "Toggle View";
+
+
```
Comment (dustinsnoap) [inline Article/Article.js]: ```//What does toggling the class "article-open" do?```

File: Menu/Menu.js [hunk 5]
```diff
@@ -33,3 +33,29 @@ let menuItems = [
...
+let header = document.querySelector('.header');
+let body = document.querySelector('body');
+let menuButton = document.querySelector('.menu-button');
+
+function menuCreator(array) {
+  let div = document.createElement('div');
+  let ul = document.createElement('ul');
+  header.appendChild(div);
+  div.classList.add('menu');
+
```
Comment (dustinsnoap) [inline Menu/Menu.js]: Good use of an array here.

**Comment by micahriley88** [comment]
> You code; as always, is very clean and easy to read. I would like to see more comments describing what things are doing, but that's about it. Good job.

Hey Dustin,
Thanks for going over my code for this project.
I didn't get to see this until after the Sprint, but coming from there I understand where you were coming from.
I went ahead and refactored my code (mostly the parameter and elements being passed through the component). I also added notations between sections to better emphasize what's going on.
I'm sending a new pull request now. If you get a chance to see this feel free to merge!
Thanks again!

**Pull request merged**